### PR TITLE
feat: introduce `router_kwargs` to customize django-ninja's `Router` attributes

### PR DESCRIPTION
**Pull request by lucasrcezimbra**
This PR enables the users to pass kwargs like `exclude_none`, `excluded_unset`, [etc](https://django-ninja.rest-framework.com/reference/operations-parameters/). to the Django Ninja Router.

### CODE REVIEW

**Review by hbakri — CHANGES_REQUESTED** [review]
Hi @lucasrcezimbra !

Firstly, I want to express my **sincere gratitude** for your interest in `django-ninja-crud` and for your valuable contribution. Having your fresh perspective on this feature is incredibly helpful and I'm really impressed with your approach to testing each `AbstractModelView` class directly. It's a fantastic addition to our test coverage!

I've reviewed your PR and overall it's excellent. However, I do have a few minor feedback points I'd like to share, and I hope we can work together to address these:

- **Test file location and naming**: For maintaining the consistency of our test structure, could you please move the test files related to views into a "tests/views" directory? Also, the file names would be more informative if they reflect what they're testing. For instance, tests for `CreateModelView` could be in a file named "test_create.py".
- **Test class naming convention**: In this project, I've been using the convention of adding "Test" as a suffix to the test classes (for example, `CreateModelViewTest`). Could we please update your new test classes to follow this convention?
- **Assertions in tests**: For the lines like: 
```python
router_mock.post.call_args[1]["exclude_none"] is True
```
I believe using an assertion like this: 
```python
self.assertIsTrue(router_mock.post.call_args[1]["exclude_none"])
``` 
would provide clearer intent and better error messaging if the test fails.
- **`router_kwargs` parameter**: I believe adding `router_kwargs` as a parameter in the `AbstractModelView` class could be beneficial as it's likely to be used by all subclasses. Here is a small snippet of what this could look like:
```python
class AbstractModelView:
    ...
    def __init__(..., router_kwargs: Optional[dict] = None):
        ...
        self.router_kwargs = router_kwargs or {}
    ...
```
Once again, I appreciate your efforts and contribution. Please let me know your thoughts on these points and if you have any questions or need any assistance implementing these changes.

**Comment by lucasrcezimbra** [comment]
> For maintaining the consistency of our test structure, could you please move the test files related to views into a "tests/views" directory?

You mean `tests/test_app/views`, `tests/views`, or `tests/test_app/tests/views`?

**Comment by lucasrcezimbra** [comment]
I think I addressed all points. Thanks for the fast review. Let me know if you find any other improvements.

File: tests/views/test_update.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views import UpdateModelView
+from tests.test_app.models import Item
+from tests.test_app.schemas import ItemIn, ItemOut
+
+
+class UpdateModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_update.py:11]: `UpdateModelViewTest`

File: tests/views/test_retrieve.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views import RetrieveModelView
+from tests.test_app.models import Item
+from tests.test_app.schemas import ItemOut
+
+
+class RetrieveModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_retrieve.py:11]: `RetrieveModelViewTest`

File: tests/views/test_patch.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views import PatchModelView
+from tests.test_app.models import Item
+from tests.test_app.schemas import ItemIn, ItemOut
+
+
+class UpdateModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_patch.py:11]: `PatchModelViewTest`

File: tests/views/test_list.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views.list import ListModelView
+from tests.test_app.models import Collection, Item
+from tests.test_app.schemas import ItemOut
+
+
+class ListModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_list.py:11]: `ListModelViewTest`

File: tests/views/test_delete.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from unittest.mock import MagicMock
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views import DeleteModelView
+from tests.test_app.models import Collection, Item
+
+
+class DeleteModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_delete.py:10]: `DeleteModelViewTest`

File: tests/views/test_create.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+from django.db import models
+from django.test import TestCase
+
+from ninja_crud.views.create import CreateModelView
+from tests.test_app.models import Collection, Item
+from tests.test_app.schemas import ItemIn, ItemOut
+
+
+class CreateModelViewSetTest(TestCase):
```
Comment (hbakri) [inline tests/views/test_create.py:11]: `CreateModelViewTest`

**Review by hbakri — COMMENTED** [review]
Thank you for the incredibly swift response and the work you've put into these changes! Everything looks almost perfect.

I've spotted a minor naming issue with the test classes, but it's a small detail. I'll point it out directly in the code using GitHub's line comment feature for a precise review.

Other than that, this is shaping up excellently. Once that little adjustment is made, I believe we'll be all set.

Again, great job, and I appreciate your collaboration on this!

File: tests/views/test_update.py (hunk 1 above)
Comment (lucasrcezimbra) [reply tests/views/test_update.py:11]:
```suggestion
class UpdateModelViewTest(TestCase):
```

File: tests/views/test_retrieve.py (hunk 2 above)
Comment (lucasrcezimbra) [reply tests/views/test_retrieve.py:11]:
```suggestion
class RetrieveModelViewTest(TestCase):
```

File: tests/views/test_patch.py (hunk 3 above)
Comment (lucasrcezimbra) [reply tests/views/test_patch.py:11]:
```suggestion
class PatchModelViewTest(TestCase):
```

File: tests/views/test_list.py (hunk 4 above)
Comment (lucasrcezimbra) [reply tests/views/test_list.py:11]:
```suggestion
class ListModelViewTest(TestCase):
```

File: tests/views/test_delete.py (hunk 5 above)
Comment (lucasrcezimbra) [reply tests/views/test_delete.py:10]:
```suggestion
class DeleteModelViewTest(TestCase):
```

File: tests/views/test_create.py (hunk 6 above)
Comment (lucasrcezimbra) [reply tests/views/test_create.py:11]:
```suggestion
class CreateModelViewTest(TestCase):
```

**Review by hbakri — APPROVED** [review]

**Pull request merged**